paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetListFragment.kt

```diff
@@ -2,20 +2,29 @@ package com.stripe.android.paymentsheet
 
 import android.os.Bundle
 import android.view.View
+import androidx.annotation.VisibleForTesting
 import androidx.appcompat.app.AppCompatActivity
-import androidx.compose.ui.graphics.toArgb
-import androidx.core.view.isVisible
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
 import androidx.fragment.app.activityViewModels
 import com.stripe.android.paymentsheet.databinding.FragmentPaymentsheetPaymentMethodsListBinding
-import com.stripe.android.ui.core.Amount
 import com.stripe.android.ui.core.CurrencyFormatter
-import com.stripe.android.ui.core.PaymentsThemeConfig
-import com.stripe.android.ui.core.isSystemDarkTheme
+import com.stripe.android.ui.core.PaymentsTheme
 
 internal class PaymentSheetListFragment() : BasePaymentMethodsListFragment(
     canClickSelectedItem = false
 ) {
-    private val currencyFormatter = CurrencyFormatter()
+    @VisibleForTesting
+    var showingTotal = false
```

Comment:
Can't we just rely on the view model for testing?

Reply:
ooops yeah. Will remove this in my next commit.